<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkPointSetInteractionControls</class>
 <widget class="QWidget" name="QmitkPointSetInteractionControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkPointSetInteractionView</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择的点集:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_CurrentPointSetLabel">
       <property name="text">
        <string>none</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_PbAddPointSet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add new point set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>当前点集</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QmitkPointListWidget" name="m_PointListWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="pelvisMarkersList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>左侧ASIS点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧ASIS点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧髋臼窝上缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧髋臼窝前缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧髋臼窝后缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧髋臼窝中心坐标</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧髋臼窝上缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧髋臼窝前缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧髋臼窝后缘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧髋臼窝中心坐标</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="femurMarkersList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>左股骨球头中心</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左小转子特征点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左大转子特征点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧内上髁</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左侧外上髁</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右股骨球头中心</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右小转子特征点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右大转子特征点</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧内上髁</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>右侧外上髁</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>股骨髓腔近端中心</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>股骨髓腔远端中心</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkPointListWidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">mitkDataNode.h</include>
  <include location="global">mitkPointSet.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
